luigi/scheduler.py

```diff
@@ -366,7 +370,12 @@ def update(self, worker_reference, get_work=False):
 
     def prune(self, config):
         # Delete workers that haven't said anything for a while (probably killed)
-        if self.last_active + config.worker_disconnect_delay < time.time():
+        if self.custom_disconnect_delay:
+            disconnect_delay = self.custom_disconnect_delay
+        else:
+            disconnect_delay = config.worker_disconnect_delay
+        if self.last_active + disconnect_delay < time.time():
+            logger.debug("Worker %s timed out (no contact for >=%ss)", self, disconnect_delay)
```

Comment:
Now that Luigi is Python 3.6+, consider using `.format()` strings or `f-strings`, instead of the legacy `%-strings`.